src/index.tsx

```diff
@@ -46,6 +46,10 @@ const dataServiceUrl = '/eda-data-service';
 const userServiceUrl = '/eda-user-service';
 const downloadServiceUrl = '/eda-user-service';
 
+// Set singleAppMode to the name of one app, if the eda should use one instance of one app only.
+// Otherwise, let singleAppMode remain undefined or set it to ''
+const singleAppMode = '';
```

Comment:
I think we should allow this to be set in `.env`, just like `exampleAnalysesAuthor` below.